nemo/collections/asr/data/audio_to_label.py

```diff
@@ -53,8 +53,9 @@ class AudioToSpeechLabelDataSet(Dataset):
             Controls whether the dataloader loads the audio signal and
             transcript or just the transcript.
             Defaults to True.
+        time_length (float): max seconds to consider in a batch # Pass this only for speaker recognition and VAD task 
```

Comment:
for VAD its not max_seconds in a batch, it is the window time length

Reply:
yes yes good catch. forget to update it